static/js/latest-news.js

```diff
@@ -0,0 +1,119 @@
+function formatDate(date) {
+  const parsedDate = new Date(date);
+  const monthNames = [
+    "January",
+    "February",
+    "March",
+    "April",
+    "May",
+    "June",
+    "July",
+    "August",
+    "September",
+    "October",
+    "November",
+    "December"
+  ];
+
+  const day = parsedDate.getDate();
+  const monthIndex = parsedDate.getMonth();
+  const year = parsedDate.getFullYear();
+
+  return day + " " + monthNames[monthIndex] + " " + year;
```

Comment:
Japanese date format (assuming monthIndex starts at 0)

```suggestion
  return year + "年" + (monthIndex + 1) + "月" + day + "日";
```